.github/ISSUE_TEMPLATE/execution_bug_report.md

```diff
@@ -0,0 +1,65 @@
+---
```

Comment:
This will be a dumping ground. 


```
---
name: Execution bug report
about: You have an issue to execute xmr-stak.

 ---

**Most execution issues are caused by driver problems. Please use the [xmr-stak sub-reddit](https://www.reddit.com/r/XmrStak/) to ask for help instead of opening an issue here.**
```